cpanfile

```diff
@@ -6,7 +6,7 @@ requires "Moo" => "0";
 requires "PPI::Document" => "0";
 requires "Path::Tiny" => "0";
 requires "Pod::Usage" => "0";
-requires "Test::Vars" => "0";
```

Comment:
I think this line is enough. We don't need to set the version in `dist.ini` and the other changes will be generated by `Dist::Zilla` as well. I could have been a lot clearer about that. My bad!

Reply:
No problem! I have reverted the other changes :)